package.json

```diff
@@ -8,7 +8,8 @@
   "main": "./ng2-smart-table.js",
   "typings": "./ng2-smart-table.d.ts",
   "dependencies": {
-    "ng2-completer": "^1.0.0"
+    "ng2-completer": "^1.0.0",
+    "selectivity": "^3.0.3"
```

Comment:
@damnko, Now, we start to use [yarn](https://yarnpkg.com/lang/en/) with yarn lock file and if you add new packages, please, use `yarn add selectivity` command in your console.